src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -513,13 +513,25 @@ public function column_source( $post ) {
 		}
 
 		$html[] = '<p>' . esc_html_x( 'via ', 'record via origin', 'the-events-calendar' ) . '<strong>' . $source_info['via']  . '</strong></p>';
+
+		$html[] = '<div class="tribe-view-links-container" style="display:flex; flex-direction:row">';
+        if ( 'scheduled' === $this->tab->get_slug() ) {
+            $filter_link = admin_url( "edit.php?post_type=tribe_events&aggregator_record={$record->id}" );
```

Comment:
:no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: a52fb47319807f4145c1750160c0107c</sub>